org.mixedrealitytoolkit.input/Tests/Runtime/InteractionModeManagerTests.cs

```diff
@@ -22,6 +22,7 @@ namespace MixedReality.Toolkit.Input.Tests
     /// </summary>
     public class InteractionModeManagerTests : BaseRuntimeInputTests
     {
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
I understand that most of this code relies on obsolete controllers.

Can you add a better description here as to what in particular is obsolete....

Reply:
Sure, no problem, done in https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/757/commits/8e57c242a798bf9c539d73a9ea57cdd8fe9fc69e , resolving.